rtsp.go

```diff
@@ -352,6 +356,49 @@ func (rc *rtspCamera) initH265(tracks media.Medias, baseURL *url.URL) (err error
 	return nil
 }
 
+// initMJPEG initializes the MJPEG decoder and sets up the client to receive JPEG frames.
+func (rc *rtspCamera) initMJPEG(tracks media.Medias, baseURL *url.URL) error {
+	var mjpegFormat *formats.MJPEG
+	track := tracks.FindFormat(&mjpegFormat)
+	if track == nil {
+		rc.logger.Warn("tracks available")
+		for _, x := range tracks {
+			rc.logger.Warnf("\t %v", x)
+		}
+		return errors.New("MJPEG track not found")
+	}
+
+	_, err := rc.client.Setup(track, baseURL, 0, 0)
+	if err != nil {
+		return err
+	}
+
+	mjpegDecoder, err := mjpegFormat.CreateDecoder2()
+	if err != nil {
+		return errors.Wrap(err, "error creating MJPEG decoder")
+	}
+
+	rc.client.OnPacketRTP(track, mjpegFormat, func(pkt *rtp.Packet) {
+		frame, _, err := mjpegDecoder.Decode(pkt)
+		if err != nil {
+			return
+		}
+		if frame == nil {
+			return
+		}
+
+		img, err := jpeg.Decode(bytes.NewReader(frame))
```

Comment:
Make sense to me to simply use image/jpeg instead of FFmpeg decode here.

Reply:
Yeah should just be JPEG frames we are dealing with